metadata-ingestion/tests/unit/sdk_v2/test_datajob.py

```diff
@@ -0,0 +1,346 @@
+import pathlib
+import re
+from datetime import datetime, timezone
+from unittest import mock
+
+import pytest
+
+from datahub.emitter.mce_builder import DEFAULT_ENV
+from datahub.emitter.mcp_builder import ContainerKey
+from datahub.errors import ItemNotFoundError
+from datahub.metadata.urns import (
+    CorpUserUrn,
+    DataJobUrn,
+    DataPlatformInstanceUrn,
+    DataPlatformUrn,
+    DomainUrn,
+    TagUrn,
+)
+from datahub.sdk.container import Container
+from datahub.sdk.dataflow import DataFlow
+from datahub.sdk.datajob import DataJob
+from datahub.testing.sdk_v2_helpers import assert_entity_golden
+
+GOLDEN_DIR = pathlib.Path(__file__).parent / "datajob_golden"
+GOLDEN_DIR.mkdir(exist_ok=True)
+
+
+def test_datajob_basic(pytestconfig: pytest.Config) -> None:
+    # Create a dataflow first
+    flow = DataFlow(
+        platform="airflow",
+        name="example_dag",
+    )
+
+    # Create a basic datajob
+    job = DataJob(
+        flow=flow,
+        name="example_task",
+    )
+
+    # Check URN setup
+    assert DataJob.get_urn_type() == DataJobUrn
+    assert isinstance(job.urn, DataJobUrn)
+    assert (
+        str(job.urn)
+        == f"urn:li:dataJob:(urn:li:dataFlow:(airflow,example_dag,{DEFAULT_ENV}),example_task)"
```

Comment:
this should just be hardcoded

```suggestion
        == "urn:li:dataJob:(urn:li:dataFlow:(airflow,example_dag,PROD),example_task)"
```